examples/Celement_with_env_1.hs

```diff
@@ -3,16 +3,19 @@ module Concept where
 import Tuura.Concept.STG
 
 -- C-element with environment circuit described using signal-level concepts
-circuit a b c = interface <> outputRise <> inputFall <> outputFall <> inputRise <> initialState
+
+data Sign = A | B | C deriving (Eq, Enum, Bounded)
```

Comment:
I'd prefer to keep the full name `Signal` to avoid confusion ("sign" is a word). It's a local name, so there shouldn't be any issues with the name clashes.

Reply:
This was just from testing, the name doesn't affect anything, but I will set it to `Signal`.